src/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  margin: 0;
+}
+
+/*header-navigation*/
+
+.header-nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
+  margin: 0 0 -4px;
+}
+
+.header-logo {
+  height: 40px;
+}
+
+.nav-menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.menu-link {
+  display: block;
+  position: relative;
+  margin-right: 20px;
+  font: 500 12px/60px Roboto, sans-serif;
+  color: #000;
+  text-decoration: none;
+}
+
+li:last-of-type .menu-link {
+  margin-right: 0;
+}
+
+.is-active,
+.menu-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  background: transparent;
+  border-radius: 10px;
+  height: 4px;
+  width: relative;
+  box-shadow: 0 55px 0 0 #00acdc;
+}
+
+/*item-card*/
+
+.card-container {
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+}
+
+.item-photo {
+  height: 134px;
+  width: 160px;
+  margin: 30px 19px;
+}
+
+.picture {
+  height: 100%;
+  width: 100%;
+}
+
+.item-heading {
+  margin: 45px 14px 5px;
+  font-weight: 500;
+  line-height: 15px;
+  font-size: 12px;
+}
+
+.item-code {
+  margin-left: 14px;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.stars-review {
+  display: flex;
+  height: 15px;
+  margin: 15px 14px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.star-container {
+  display: flex;
+  justify-content: between;
+}
+
+.star {
+  width: 13px;
+  height: 14px;
+  margin: 1px 3px 3px;
+}
+
+.star:first-of-type {
+  margin-left: 1px;
+}
+
+.response {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
```

Comment:
You repeat these styles. Place them to the container.

Reply:
I deleted those styles and removed them to container. Thanks a lot for review!